test/functional/test_framework/p2p.py

```diff
@@ -580,13 +580,13 @@ def on_version(self, message):
 
     # Connection helper methods
 
-    def wait_until(self, test_function_in, *, timeout=60, check_connected=True):
+    def wait_until(self, test_function_in, *, timeout=60, check_connected=True, delay=0.05):
```

Comment:
maybe rename "delay" to "check_interval" or another similar term?

Reply:
renamed to `check_interval`